src/org/labkey/test/components/react/ReactCheckBox.java

```diff
@@ -17,6 +19,12 @@ public ReactCheckBox(WebElement element)
     {
        super(element);
     }
+    
+    public ReactCheckBox(SearchContext context, String labelText)
```

Comment:
Since you're touching all of the usages of this class, could you remove the intercapping to match other checkbox components.
```suggestion
    public ReactCheckbox(SearchContext context, String labelText)
```

Reply:
I ended up reverting my changes to this class and removing all new usages of it. Instead I have the tests using plain old Checkbox. Since I'm no longer touching this file or adding usages I am going to skip renaming this class to keep the impact of these PRs smaller.